docs/migrations.md

```diff
@@ -124,6 +124,10 @@ class Migration(migrations.Migration):
 The `forwards()` and `backwards()` functions are where any changes
 that need to happen to a model's data are made.
 
+!!! note
+    While backwards migrations are necessary in external libraries that we create, 
+    we do not require them in cfgov-refresh. 
```

Comment:
Would it be worth adding more in the way of explanation, something like

...we do not require them in cfgov-refresh because we prefer not to rollback migrations that have already been applied.